blueprints/empress-blog-casper-template/index.js

```diff
@@ -15,14 +15,23 @@ module.exports = {
       ]
     });
 
-    applyConfig(this.project, 'responsive-image', {
-      sourceDir: 'images',
-      destinationDir: 'responsive-images',
-      quality: 80,
-      supportedWidths: [2000, 1000, 600, 300],
-      removeSourceDir: false,
-      justCopy: false,
-      extensions: ['jpg', 'jpeg', 'png', 'gif'],
+    applyBuildConfig(this.project, 'responsive-image', {
```

Comment:
Nice work figuring this part out!

Reply:
I think in the future we should figure some of the numbers here and see if we would want to change add or remove some of them